web/packages/selfhosted/test/polyfill/classic_frames_injected/test.ts

```diff
@@ -7,7 +7,7 @@ use(chaiHtml);
 
 describe("Flash inside frame with injected ruffle", () => {
     it("loads the test", async () => {
-        await openTest(browser, import.meta.dirname);
+        await openTest(browser, `polyfill/classic_frames_injected`);
```

Comment:
Maaaybe - but then we're hardcoding that things are two layers deep, what if we go three or four? 😅 For ExternalInterface locally I was originally going to have a bunch of tests in that subfolder before realizing that I only need one, but there's absolutely going to be cases in the future where we'd want more I think

Reply:
That's not what the 2 means there, it means limit of 2 (so before /test/ and after test). So it's the full directory after /test/.

Reply:
I guess this would break running tests locally if Ruffle's ancestor folder happened to be called /test/ though, huh? So maybe hard-coding does make more sense, though that circumstance is admittedly far-fetched.

Reply:
I think I'm going to stick with it as it is in this PR, at least for now. We can come back to it later - I want to give more love to tests, and maybe we can make polyfill tests a little easier to setup in general.